test/components/date-range-picker.spec.js

```diff
@@ -248,26 +249,6 @@ describe('DateRangePicker', () => {
     });
   });
 
-  describe('placeholder prop', () => {
```

Comment:
Why are the placeholder prop tests removed? The placeholder prop is still there and there's a new label prop. I know the styling is quite different, though.

Reply:
added back first test for placeholder and added both tests for label prop.